packages/nextjs/src/config/webpack.ts

```diff
@@ -73,6 +74,34 @@ export function constructWebpackConfigFunction(
       };
     }
 
+    // The SDK uses syntax (ES6 and ES6+ features like object spread) which isn't supported by older browsers. For users
+    // who want to support such browsers, `transpileClientSDK` allows them to force the SDK code to go through the same
+    // transpilation that their code goes through. We don't turn this on by default because it increases bundle size
+    // fairly massively.
+    if (!isServer && userNextConfig.sentry?.transpileClientSDK) {
+      // Find all loaders which apply transpilation to user code
+      const transpilationRules = findTranspilationRules(newConfig.module?.rules, projectDir);
+
+      // For each matching rule, wrap its `exclude` function so that it won't exclude SDK files, even though they're in
+      // `node_modules` (which is otherwise excluded)
+      transpilationRules.forEach(rule => {
+        // All matching rules will necessarily have an `exclude` property, but this keeps TS happy
+        if (rule.exclude && typeof rule.exclude === 'function') {
+          const origExclude = rule.exclude;
+
+          const newExclude = (filepath: string): boolean => {
+            if (filepath.includes('@sentry')) {
```

Comment:
I'm totalllllly not good at webpack stuff so this is probably off but: Can there be a case where a `rule` doesn't have a `exclude`? In this case should we add exclude to it?

Reply:
In webpack in general, yes, `exclude` is optional, as you'll see I've made it in the `WebpackModuleRule` type I added. But in the case of the specific rules nextjs uses for transpilation, it's always there (and has been, in all nextjs versions we support).

Next 10:
Rule using exclude function: https://github.com/vercel/next.js/blob/4f32a03ffa4902a91292d2266ec5f73b8e2b5384/packages/next/build/webpack-config.ts#L981-L999

Next 11:
Rule using exclude function: https://github.com/vercel/next.js/blob/75b7a57e0f0044d9315eb6adbd4231b67799d0b1/packages/next/build/webpack-config.ts#L1121-L1139

Next 12:
Exclude function: https://github.com/vercel/next.js/blob/cee2cf379dfe87ff7a9c9fb88fea4acbfe4a20bc/packages/next/build/webpack-config.ts#L957-L969
Rules where it's used: https://github.com/vercel/next.js/blob/cee2cf379dfe87ff7a9c9fb88fea4acbfe4a20bc/packages/next/build/webpack-config.ts#L1350-L1375